Simplenote/build.gradle

```diff
@@ -36,7 +37,13 @@ android {
         minSdkVersion 23
         targetSdkVersion 29
 
-        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
+        testInstrumentationRunner 'com.automattic.simplenote.SimplenoteAppRunner'
+    }
+
+    // Avoid duplicate annotations dependencies
+    configurations {
+        cleanedAnnotations
+        compile.exclude group: 'org.jetbrains' , module:'annotations'
```

Comment:
Suggestion (💡): This is indeed unfortunate that we need that here, probably a library is conflicting. I think this needs to be resolved on a library level. I am not sure if we really need to take a look at it now or later, but I'll definitely add a task for later, just to make sure we don't forget to look into that again at the end. Wdyt?

PS: I usually suggest atomic commits and in this case I would have added this configuration as a separate commit, which I would explain within the why it is actually needed and what it solves.

Reply:
I found the problem in tracks and Kotlin. I could do this and the error disappears:
```gradle
implementation('com.automattic:tracks:1.2') {
    exclude group: "org.jetbrains", module: "annotations-java5"
}
```

But I am not sure the behavior of tracks will remain the same.

Reply:
👋 @dandominguez04 !

Yes, I think this is a much better alternative since it targets the problems explicitly. Thanks for finding out the conflicting dependency. 🙏 

> But I am not sure the behavior of tracks will remain the same.

I believe it will, I am 99.9% sure of it, but you can never be absolutely sure with anything, can you... 😅

Reply:
Yes, I also will test this manually. I have applied the changed in 72285a54.

Reply:
Great, LGTM! 👍